curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/iterate-with-javascript-for-loops.english.md

```diff
@@ -52,13 +52,6 @@ tests:
 <div id='js-seed'>
 
 ```js
-// Example
```

Comment:
I tried to update the solution sections, but I may have missed some.  I think I would rather create a separate PR that removes all comments from the solutions.  What do you think?

Reply:
That's fine, as long as we know what not to look for.  By the way, I'm leaving this conversation unresolved, because resolving it hides the conversation before the other person has a chance to read it.  I do think it's resolved, though!